debug/cellect.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_record'
+require 'cellect/server'
+require_relative '../lib/cellect/server/adapters/panoptes'
+require 'otr-activerecord'
+require 'pry'
+
+OTR::ActiveRecord.configure_from_url! ENV['DATABASE_URL']
+
+#ActiveRecord::Base.connection_config
```

Comment:
Commented out? Did you mean for this to remain like this here? If you want it for future reference, maybe we could have it behind an `if ENV["DATABASE_CONNECT"]` or something?